spec/std/socket_spec.cr

```diff
@@ -169,15 +169,67 @@ describe Socket::IPAddress do
 end
 
 describe Socket::UNIXAddress do
+  {% if flag?(:linux) %}
+    it "can be abstract on Linux" do
+      path = "/abstract.sock"
+      addr = Socket::UNIXAddress.new('@' + path)
+      addr.abstract?.should be_true
+      addr.path.should eq(path)
+    end
+  {% else %}
+    it "raises for abstract UNIX address on non-Linux" do
+      expect_raises(ArgumentError, /not supported/) do
+        Socket::UNIXAddress.new("@/abstract.sock")
+      end
+    end
+
+    it "is not abstract on non-Linux" do
```

Comment:
Is there a reason this example should not run on linux target?

Reply:
no indeed